scripts/openyurt_deployer/node/demo.go

```diff
@@ -0,0 +1,130 @@
+
+	createCloudNpTemplate := template.CreateCloudNpTemplate()
+	logs.WaitPrintf("Creating yaml files for cloud nodepool")
+	_, err = masterNode.ExecShellCmd(createCloudNpTemplate, cloudPoolName, cloudFile)
+	logs.CheckErrorWithTagAndMsg(err, "Failed to create yaml for cloud\n")
+
+	// edge.yaml
+	createEdgeNpTemplate := template.CreateEdgeNpTemplate()
+	logs.WaitPrintf("Creating yaml files for edge nodepool")
+	_, err = masterNode.ExecShellCmd(createEdgeNpTemplate,edgePoolName, edgeFile)
+	logs.CheckErrorWithTagAndMsg(err, "Failed to create yaml for edge\n")
+
+	//label master as cloud TODO not just master, but all cloud nodes
+	logs.WaitPrintf("Labeling master")
+	_, err = masterNode.ExecShellCmd(`kubectl label node %s apps.openyurt.io/desired-nodepool=%s`, masterNode.Configs.System.NodeHostName, cloudPoolName)
+	logs.CheckErrorWithTagAndMsg(err, "Master Cloud label fail\n")
+
+	//label edge
+	logs.WaitPrintf("Labeling workers")
+	for _, worker := range workerNodes {
+		worker.GetNodeHostName()
+		var desiredNpName string
+		if worker.NodeRole == "cloud"{
+			desiredNpName = cloudPoolName
+		}else{
+			desiredNpName = edgePoolName
+		}
+		_, err = masterNode.ExecShellCmd("kubectl label node %s apps.openyurt.io/desired-nodepool=%s", worker.Configs.System.NodeHostName, desiredNpName)
+		logs.CheckErrorWithTagAndMsg(err, "worker label fail\n")
+	}
+	logs.SuccessPrintf("Label success\n")
+
+	logs.WaitPrintf("Apply cloud.yaml")
+	_, err = masterNode.ExecShellCmd("kubectl apply -f %s", cloudFile)
+	logs.CheckErrorWithTagAndMsg(err, "Failed to apply cloud.yaml\n")
+
+	logs.WaitPrintf("Apply edge.yaml")
+	_, err = masterNode.ExecShellCmd("kubectl apply -f %s", edgeFile)
+	logs.CheckErrorWithTagAndMsg(err, "Failed to apply edge.yaml\n")
+	
+	// Create yurt app set
+	createYurtTemplate := template.GetYurtAppSetTemplate()
+	logs.WaitPrintf("Create yurt app set yaml file")
+	_, err = masterNode.ExecShellCmd(createYurtTemplate, cloudPoolName, cloudPoolName, edgePoolName, edgePoolName, yurtFile)
+	logs.CheckErrorWithTagAndMsg(err, "Failed to create yurt app set yaml\n")	
+	
+	logs.WaitPrintf("Apply yurt app set yaml file")
+	_, err = masterNode.ExecShellCmd("kubectl apply -f %s", yurtFile)
+	logs.CheckErrorWithTagAndMsg(err, "Failed to apply yurt app set yaml\n")
+
+	out, err := masterNode.ExecShellCmd("kubectl get deployments | awk 'NR>1 {print $1}'")
+	deployments := strings.Split(out, "\n")
+	fmt.Printf("NodePool Information:\n")
+	fmt.Printf("+--------------------------------------------------------------------+\n")
+	fmt.Printf("+Cloud Nodepool %s:\n", cloudPoolName)
+	fmt.Printf("+\t\t|_deployment: %s \n", deployments[0])
+	fmt.Printf("+\tnode: %s <- Master\n", masterNode.Configs.System.NodeHostName)
+	for _, worker := range workerNodes {
+		if worker.NodeRole == "cloud"{
+			fmt.Printf("+\tnode: %s\n", worker.Configs.System.NodeHostName)
+		}
+	}
+
+	fmt.Printf("+Edge  Nodepool %s:\n", edgePoolName)
```

Comment:
don't use `fmt`, use the logger library from Haoyuan (or the generic logger we use in the core vHive)